src/main/java/thePackmaster/actions/siegepack/DigInAction.java

```diff
@@ -0,0 +1,34 @@
+package thePackmaster.actions.siegepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.util.Wiz;
+
+import static com.megacrit.cardcrawl.dungeons.AbstractDungeon.player;
+
+public class DigInAction extends AbstractGameAction {
+    private final int ENERGY_GAIN;
+    AbstractPower flashingBuff;
+
+    public DigInAction(int gain, AbstractPower source) {
+        duration = 0.25F;   //Should be short enough that player can't have played any block yet.
```

Comment:
Nothing else will be happening during this action anyway, there's no reason for this to not be a singleshot action that just sets isDone to true